components/style/servo/selector_parser.rs

```diff
@@ -430,11 +430,11 @@ impl<'a> ::selectors::Parser for SelectorParser<'a> {
     }
 
     fn default_namespace(&self) -> Option<Namespace> {
-        self.namespaces.default.clone()
+        self.namespaces.default.clone().map(|(ns, _)| ns)
```

Comment:
Instead of cloning both components and getting only the first one, perhaps it's worth to do it the other way around?